site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -35,25 +38,24 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 ==== Request Body
 
 [.api]
-[field]#displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
-A human readable name for this link. This value should be used to make it easier to identify the user this link represents in the remote identity provider.
+[field]#identityProviderLink.displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
+A human-readable value identifying the user in the remote identity provider.
 +
-This value is optional and it will always be set by FusionAuth the next time this link is used to resolve the FusionAuth user during a login event for this IdP.
+After the link is created, and a user successfully logs in for the first time with the IdP, this value will typically be set to an email or username.
```

Comment:
I think you lost the context of this comment on the re-write. The original comment was describing what happens when the value is omitted because it is optional. Can you also verify the previous comment was correct, do we set it if not provided?

Reply:
Based on the code path for the Google Identity Provider, I would say this is correct that FusionAuth will resolve it the next time a user logs in. Based on here: io/fusionauth/api/service/authentication/GoogleIdentityProviderAuthenticationService.java:82 and here io/fusionauth/api/service/authentication/BaseIdentityProviderAuthenticationService.java:648, it seems we are either using the email or username returned on the id_token returned from Google Login and then updating or creating an existing link. After manually creating a link, and then logging in, I now see two Google Links on my user account. I will write a test to see if this is an error on my part or a small bug in the way we update a link. leaving my draft description below until I finish testing.

This value is optional and it will always be set by FusionAuth the next time this link is used to resolve the FusionAuth user during a login event for this IdP. This value will typically be set to an email or username when the user is resolved, depending on Identity Provider and the linking strategy.

Reply:
I was able to get the link to update with the correct `displayName`. Since my Google JWT was scoped for email, and I was using link on email, the displayName did update to my email address. 👍

Reply:
I think the original description is better:

A human-readable name for this link for identification and convenience. This value will generally be an email address or username. This value can be set by the API caller. So for instance, this value could be `Initial piedpiper link from user migration`. This value will be updated during the next login for the linked user based on the identity provider (`richard@piedpiper.com`, for instance). This value may not necessarily reflect the username or email you used to authenticate with the 3rd party identity provider.

Reply:
My edited version of the original:

A human-readable name to identify the link. This value will be updated during the next login for the linked user based on the linking strategy for the Identity Provider and the values returned by the Identity Provider. This value may not necessarily reflect the username or email you used to authenticate with the linked Identity Provider.